src/components/labTabsComponent/TabsAndButtons.jsx

```diff
@@ -41,9 +42,12 @@ export default function TabsAndButtons() {
     if (result) {
       dispatch(updateTabArray(result));
     }
-    if (selectedTab) {
+    if (selectedTab || selectedTab === 0) {
       setValue(selectedTab);
       dispatch(changeSelectedTabId(tabsArray[selectedTab]?.id));
+    } else {
+      setValue(0);
+      dispatch(changeSelectedTabId(0));
```

Comment:
if you store this data in redux, don't use the local store too. Instead get data from  redux